### DISCUSSION TITLE
유니티 완전완전 초보 [제페토 가이드 설명만 읽고도 만들 수 있나요..?]

### DISCUSSION THREAD

**Discussion by almondsky** [naverz/zepeto-studio-kor#754] [4. General]
안녕하세요, 
제목처럼.. 가나다라 부터 배워야하는 유니티 완전완전 초보입니다. 제페토 월드를 만들고 싶은데,, 제페토 가이드라인에 나와있는 설명들만 읽고도 만들 수 있나요..? 아니면 따로 유니티 관련 수업을 학원이나 국비 수업을 통해 들어야 가능할까요..? 고수님들 답변 주시면 매우 감사하겠습니다.! 꾸벅!

**Comment by x4cf** [comment]
프로그래밍이나 개발 자체가 처음이시라면 가이드만 보고는 조금 어려울 수 있습니다.

수업을 통해 유니티 개발을 배우고 진입해도 좋지만, 아직 제페토 뉴 월드를 교육하는 기관은 없습니다.
제페토 공식 가이드 또는 유저 자료를 이용하는것도 좋은 방법입니다 :D 

https://github.com/naverz/zepeto-studio-kor/discussions/708#discussioncomment-2704309

**Reply by almondsky** [reply]
답변 너무 감사합니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 난이도 별로 제작된 샘플 월드를 따라서 만들어 보시기를 추천 드립니다.
관련해서 공식 가이드 영상도 제공될 예정이오니 조금만 기다려 주세요!

https://github.com/naverz/zepeto-studio-kor/discussions/615

감사합니다.

**Reply by almondsky** [reply]
답변 너무 감사합니다!
